test/unit/convert.test.js

```diff
@@ -42,6 +42,10 @@ describe('convert', () => {
   createFixtureTest('tables', { type: 'copy' });
   createFixtureTest('tables', { type: 'export', skip: true });
 
+  // TODO: utilize internal clipboard data in tests, then enable these tests.
+  // createFixtureTest('internal-links', { type: 'copy' });
+  // createFixtureTest('internal-links', { type: 'export', skip: true });
```

Comment:
I added fixture file support for the GDocs internal copy format in #111, so it should be pretty straightforward to implement this test now.

Reply:
I went ahead and rebased this to get the new fixtures and added support for them to `createFixtureTest()`. It’s a good thing, because after uncommenting, this test doesn’t actually pass!

I’ll have to do some more overall cleanup on this PR later in the week.